### DISCUSSION TITLE
there are news??

### DISCUSSION THREAD

**Discussion by alessandro315** [omersiar/esp-rfid#481] [General]
Is coming soon news for this wonderful porject?? 
with the expected compatibility with esp32?? 
ale :)

**Comment by matjack1** [comment]
Hey @alessandro315 :)

There are a lot of news! We are doing many little things on the project, everything on the dev branch.

Hopefully we'll release a new stable version soon.

For the ESP32 porting instead personally I have no plans and I'm not sure about the effort. What's your motivation to support also the ESP32?

**Comment by Leissson** [comment]
I have noticed this activity. Joined Github and bought Official ESP-RFID Relay Board just because great effort have been made lately by @matjack1 . Absolutely brilliant job and can't wait to install upcoming stable version with keypad support. Many thanks to all contributors. Keep up the good work!

**Reply by matjack1** [reply]
Thanks! You should be able to install the dev branch on the board, have you tried already?

You can get a quick image from here: https://github.com/esprfid/esp-rfid/actions/runs/1795525934 :)

**Reply by Leissson** [reply]
Thank you, still waiting parcel to arrive. But I'll give it a try.

**Comment by alessandro315** [comment]
I have no need for implementation on esp32, I thought they were compatibility problems with the small and very sweet esp8266. I have problems with the keypads of the rfid-reader.
This rfid reader: [https://www.amazon.it/Retekess-Controllo-Retroilluminata-Antipolvere-Magazzino/dp/B07CYNLF96/ref=sr_1_1?adgrpid=123534019160&gclid=Cj0KCQiAjJOQBhCkARIsAEKMtO0mO8yJuNDa5drCZw9nH7W1KmjntlEp9PegdCdInBmI5o8KCrO7tP8aAp0cEALw_wcB&hvadid=515113205518&hvdev=c&hvlocphy=9050696&hvnetw=g&hvqmt=e&hvrand=305228093513565968&hvtargid=kwd-1007135839325&hydadcr=12405_1852838&keywords=retekess+t-ac04&qid=1644517157&sr=8-1](url)
Not seeing people with the same problem around I thought they were compability problems.
But reading I learned that it will be a future implementation of the keypads ?? Right??
however a very respectable project, very good!
ale :+1:

**Comment by alessandro315** [comment]
I also wanted to ask but the problem with the number pad, I only have them?
I use wiegand as a protocol, with rfid I have no problems, but with the keypad nothing to do.
could you help me in case you know more.
With arduino and quest schetck: https://github.com/nygma2004/RFID_Wiegand_MQTT it works great.
i would like to use esp-rfid because i'm loving this project, i saw that there is an official relay board, if things definitely improve it will be my next purchase.
ale 👍

**Comment by alessandro315** [comment]
I just installed the dev version, you guys are great, I had problems with the logs, with the webserver, with the html page, with mqtt, with the keypads of the rfid-reader.

All solved. If I had contacted you a few days ago, I would have gone mad a little less.
You are great.
The best thing is the implementation of Home Assistant, everything is ready.

A beautiful job I would love to learn how to solve problems, as you do, if you can give me some advice I would be very happy.

ale 👍

**Reply by matjack1** [reply]
so it's all working fine in the dev version? Or do you still have some problems?

**Reply by alessandro315** [reply]
hey Mat, after days of testing almost no problem. just an anomalous block in all this time, I don't know what caused it. And my esp when I'm in the log screen almost always restarts, for no reason. these are the problems encountered after days of testing. I hope I was helpful, any questions and tests, happy to deal with it.
Ale :)

**Reply by matjack1** [reply]
ok @alessandro315 meanwhile I've added this PR: https://github.com/esprfid/esp-rfid/pull/484 that should reduce the anomalous blocks. Would you mind giving this a go and let me know if it improve things? Thank you!!
